### ISSUE TITLE
If GOOGLE_APPLICATION_CREDENTIALS, check that JSON file exists

### ISSUE THREAD

**Issue by nick-youngblut** [nextflow-io/nextflow#4895]
## New feature

Currently, if the user sets the `GOOGLE_APPLICATION_CREDENTIALS` environmental variable to use GCP Batch for running jobs with a nextflow pipeline, and `GOOGLE_APPLICATION_CREDENTIALS` is set to the wrong path (the service key json file does not actually exist at that path), the the Nextflow pipeline will still run, but it will generate errors, such as `ERROR ~ 401 Unauthorized`. 

It would be helpful if the Nextflow Google plugin checked whether the JSON service key file exists, if the user has set the `GOOGLE_APPLICATION_CREDENTIALS` environmental variable. 

## Usage scenario 

Better debugging of setup issues (and thus less posts on the Nextflow Slack workspace). 

## Suggest implementation 

A simple `if GOOGLE_APPLICATION_CREDENTIALS env var is set, does the path point to a file (and is that file a valid json)?`

**Comment by nick-youngblut** [comment]
Any updates on this? I just ran into this issue again recently.

**Comment by pditommaso** [comment]
How to replicate this issue? In my test, when the credentials path the execution is  stopped with this error message 

```
Invalid or corrupted Gogole credentials file: /som/creds/path 
```

**Comment by pditommaso** [comment]
Yep, that was addressed too 👉 1b0d4525
